semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -153,16 +153,16 @@ let variance_modifier (env : env) (x : CST.variance_modifier) =
 
 let member_modifier (env : env) (x : CST.member_modifier) =
   (match x with
-   | `Over tok -> token env tok (* "override" *)
-   | `Late tok -> token env tok (* "lateinit" *)
+   | `Over tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "override" *)
+   | `Late tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "lateinit" *)
   )
 
 let class_modifier (env : env) (x : CST.class_modifier) =
   (match x with
-   | `Sealed tok -> token env tok (* "sealed" *)
-   | `Anno tok -> token env tok (* "annotation" *)
-   | `Data tok -> token env tok (* "data" *)
-   | `Inner tok -> token env tok (* "inner" *)
+   | `Sealed tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "sealed" *)
```

Comment:
What is the difference between sealed and final in Java?
Maybe we can generate a Final here?

Reply:
I think the sealed class is more of an enum class that you can have multiple instances of ("restricted class hierarchy where an object or a value can have one of the types from a limited set of values" from https://kotlinlang.org/), so its not the csharp sealed class 😓 

So sealed classes are different from final classes 😮